### ISSUE TITLE
teamviewer@15.27.3: hash check failed

### ISSUE THREAD

**Issue by JuanMM94** [ScoopInstaller/Extras#8460]
#### Console Output
```
Installing 'teamviewer' (15.27.3) [64bit]
TeamViewerPortable.zip (40.5 MB) [============================================================================] 100%
Checking hash of TeamViewerPortable.zip ... ERROR Hash check failed!
App:         extras/teamviewer
URL:         https://download.teamviewer.com/download/version_15x/TeamViewerPortable.zip
First bytes: 50 4B 03 04 0A 00 02 00
Expected:    407d7dcfa113860759b08a774b0ee8275c38d5bf0694da6a042aa0e9954eb993
Actual:      a63cd4bb8ac75957b58b55d90ec1ba607149d15a941b9d08ab9fd2143db7d85d
```

#### Scoop Configuration

```json
{
  "lastupdate": "2022-05-11T13:32:56.2324673-03:00",
  "SCOOP_REPO": "https://github.com/ScoopInstaller/Scoop",
  "SCOOP_BRANCH": "master"
}
```

#8201 doesn't work.

**Comment by JuanMM94** [comment]
> You reported version `15.27.3`, but the latest available version is `15.29.4`.
> 
> Run `scoop update; scoop update teamviewer --force`

This is not a valid response, as I stated on the original comment.

**Comment by JuanMM94** [comment]
@rashil2000 Could you please help me? Thanks.
